### PR TITLE
Electron debugging support

### PR DESCRIPTION
**Pull request by RodBerw**

### CODE REVIEW

**Comment by EzioLi01** [comment]
Please share a sample of debug configuration format.

**Comment by RodBerw** [comment]
> Please share a sample of debug configuration format.

Make sure to add both 'electron' and 'browser' as new platforms in cordova with command 'cordova platform add", also you'll need to install electron through 'npm i electron' to get it's executable.

You can add the debug config format by here:
image
image

**Comment by EzioLi01** [comment]
Could you please add ` - Experimental` tag in config name field?

**Comment by EzioLi01** [comment]
> Experimental

Please change tag with the uppercase 'E'.

File: package.json [hunk 1]
```diff
@@ -1,756 +1,777 @@
...
+                            "target": "chrome",
+                            "simulatePort": 8000,
+                            "livereload": true,
+                            "cwd": "^\"\\${workspaceFolder}\"",
+                            "sourceMaps": true
+                        }
+                    },
+                    {
+                        "label": "Cordova: Debug on Electron - experimental",
+                        "description": "%cordova.snippets.simulateElectron%",
```
Comment (EzioLi01) [inline package.json:338]: Have you added english description for it?

**Comment by EzioLi01** [comment]
Also, please check package.json file format, why all file format has been changed?

File: package.json (hunk 1 above)
Comment (RodBerw) [reply package.json:338]: Added to the new commit!

**Comment by RodBerw** [comment]
> > Experimental
> 
> Please change tag with the uppercase 'E'.

Sorry, I tried to follow others configs model. Already fixed!

**Comment by RodBerw** [comment]
> Also, please check package.json file format, why all file format has been changed?

Strange, only the modified lines appear on source control diffs in my vscode, not all the file. However all the file format seems to be formatted here by some reason... The file content seems to be intact, but I could try to commit it again and investigate the reason for this.

File: src/debugger/requestArgs.ts [hunk 2]
```diff
@@ -31,6 +31,9 @@ export interface ICordovaAttachRequestArgs extends DebugProtocol.AttachRequestAr
     runtimeVersion?: string;
     hostname?: string;
 
+    //Electron debug properties
```
Comment (EzioLi01) [inline src/debugger/requestArgs.ts:34]: Missing a space.

**Comment by RodBerw** [comment]
I updated code indentation to fix format issues.

**Review by EzioLi01 — APPROVED** [review]

**Pull request merged**